spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/RewriteDataFilesSparkAction.java

```diff
@@ -331,24 +331,31 @@ private Result doExecuteWithPartialProgress(
       RewriteDataFilesCommitManager commitManager) {
     ExecutorService rewriteService = rewriteService();
 
-    // Start Commit Service
+    // start commit service
     int groupsPerCommit = IntMath.divide(ctx.totalGroupCount(), maxCommits, RoundingMode.CEILING);
     RewriteDataFilesCommitManager.CommitService commitService =
         commitManager.service(groupsPerCommit);
     commitService.start();
 
-    // Start rewrite tasks
+    Collection<FileGroupFailureResult> rewriteFailures = new ConcurrentLinkedQueue<>();
```

Comment:
Looks like we are also backporting https://github.com/apache/iceberg/pull/7361 to Spark 3.3?  No problem with me, but we may want to note that in pr description in case some issues arise because of this.